src/app/loadout-builder/generated-sets/GeneratedSetButtons.m.scss

```diff
@@ -1,6 +1,18 @@
 .buttons {
+  text-align: center;
+  margin-right: 14px;
+  margin-bottom: 14px;
+
+  // Matches GeneratedSetItem width
+  width: calc(var(--item-size) + ((0.59 * var(--item-size)) * 2) + 10px);
+
   > * {
-    margin-left: 4px;
+    // Smallest width for en:"Compare Loadout" string to be a single line
+    min-width: 121px;
```

Comment:
Since this is localized, it's generally better to just do `white-space: nowrap`

Reply:
Yeah hard-coding the width was a terrible idea. Longer locale strings break the layout when `white-space: nowrap` is used though, so I've adjusted it to accept that sometimes the buttons will wrap to a taller layout